Examples/ArcGISToolkitExamples/Misc/ARStatusViewController.swift

```diff
@@ -97,6 +115,39 @@ class ARStatusViewController: UITableViewController {
         }
     }
 
+    /// The horizontal accuracy of the last location.
+    public var horizontalAccuracy: Double = 1.0 {
```

Comment:
If you wanted to get fancy, you could make these measurements:

```swift
public var horizontalAccuracyMeasurement = Measurement(value: 1, unit: UnitLength.meters)
```

And then use a measurement formatter to convert the measurement to a string with the appropriate units. That's typically how we do this sort of thing in the samples app (at least for more recent samples).